fuzzing/config/config_defaults.go

```diff
@@ -64,6 +64,9 @@ func GetDefaultProjectConfig(platform string) (*ProjectConfig, error) {
 				TraceAll:                     false,
 				TargetFunctionSignatures:     []string{},
 				ExcludeFunctionSignatures:    []string{},
+				ReversionMeasurement: ReversionMeasurementConfig{
+					Enabled: true,
```

Comment:
```suggestion
					Enabled: false,
```
I'm hesistant to leave this enabled by default. Can you also update the [docs](https://github.com/crytic/medusa/blob/master/docs/src/project_configuration/fuzzing_config.md)?